extensions/community/AudioByName.json

```diff
@@ -0,0 +1,208 @@
+            "    eventsFunctionContext.getArgument(\"Volume\"),",
+            "    eventsFunctionContext.getArgument(\"Pitch\"));",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Audio File Resource Name",
+          "longDescription": "The final string must match the full resource name as displayed in the resource list.",
+          "name": "Music",
+          "type": "string"
+        },
+        {
+          "description": "Channel identifier",
+          "name": "Channel",
+          "type": "expression"
+        },
+        {
+          "defaultValue": "no",
+          "description": "Repeat the sound",
+          "name": "Loop",
+          "optional": true,
+          "type": "yesorno"
+        },
+        {
+          "description": "Volume",
+          "longDescription": "From 0-100, with 100 being full volume.",
+          "name": "Volume",
+          "type": "expression"
+        },
+        {
+          "description": "Pitch (speed)",
+          "longDescription": "1 for normal speed",
+          "name": "Pitch",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Play a sound file on a specific channel by name or string expression. \nThis can be useful to reduce the number of events needed to dynamically play different sound files. e.g. Playing a voice over line based off the current dialogue node name.\n\nNote: Sound files should be preloaded before using these actions, such at the start of the first scene, current scene, or in the resources panel for the entire game. This is to avoid any potential audio pause since the normal resource loading at the start of the game or scene won't load these files by default.",
+      "fullName": "Play sound on a channel by name",
+      "functionType": "Action",
+      "name": "PlaySoundOnChannelByName",
+      "sentence": "Play the sound _PARAM1_ on channel _PARAM2_, loop: _PARAM3_, volume: _PARAM4_, pitch: _PARAM5_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "const soundManager = runtimeScene.getGame().getSoundManager();",
+            "soundManager.playSoundOnChannel(",
+            "    eventsFunctionContext.getArgument(\"Sound\"),",
+            "    eventsFunctionContext.getArgument(\"Channel\"),",
+            "    eventsFunctionContext.getArgument(\"Loop\"),",
+            "    eventsFunctionContext.getArgument(\"Volume\"),",
+            "    eventsFunctionContext.getArgument(\"Pitch\"));",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Audio File Resource Name",
```

Comment:
```suggestion
          "description": "Audio file resource name",
```

Reply:
I *just* caught this, didn't get to update it in time. Thanks for checking. Uploading now.